src/main/java/org/example/api/DeliveryEmployeeService.java

```diff
@@ -29,4 +29,18 @@ public int createDeliveryEmployee(DeliveryEmployeeRequest employee) throws Proje
             return id;
 
     }
+
+    public DeliveryEmployee getDeliveryEmployeeById (int id) throws SQLException, ProjectException {
+        try {
+            DeliveryEmployee deliveryEmployee = deliveryEmployeeDao.getDeliveryEmployeeById(id);
+
+            if(deliveryEmployee == null) {
+                throw new ProjectException();
+            }
+            return deliveryEmployee;
+        } catch (SQLException e) {
+            System.err.println(e.getMessage());
+            throw new ProjectException();
```

Comment:
you shouldn't catch SQLException here, throw it so the controller can return a 500